leaderelection/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>io.scalecube</groupId>
+		<artifactId>scalecube-parent</artifactId>
+		<version>0.9.1-SNAPSHOT</version>
+	</parent>
+	<artifactId>scalecube-leaderelection</artifactId>
+	<name>ScaleCube/LeaderElection</name>
+
+	<dependencies>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>scalecube-cluster</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.unquietcode.tools.jstate/jstate -->
+		<dependency>
+			<groupId>com.unquietcode.tools.jstate</groupId>
```

Comment:
The general philosophy is to use as less as possible dependencies. Also regards this specific dependency it seems that it is pretty raw and not really much supported. Also we are using it in a pretty trivial use case, so I propose to code the logic instead of using third party.